tests/js-unit/dash_table_queries_test.ts

```diff
@@ -265,6 +265,20 @@ describe('Dash Table Queries', () => {
                                 target: {a: '\r\n1\r\n'},
                                 valid: true,
                                 evaluate: true
+                            },
+                            {
+                                name: "compare 'abc' to 'ABC' (insensitive)",
+                                query: `${c.hideOperand ? '' : '{a} '}ieq ABC`,
+                                target: {a: 'abc'},
+                                valid: true,
+                                evaluate: true
+                            },
+                            {
+                                name: "compare 'abc' to 'ABC' (sensitive)",
+                                query: `${c.hideOperand ? '' : '{a} '}seq ABC`,
+                                target: {a: 'abc'},
+                                valid: true,
+                                evaluate: false
```

Comment:
Multiple new tests for the (in)sensitive versions of the operators -- as used by the table itself